concurrent_test.go

```diff
@@ -511,6 +537,23 @@ func (rs historyRecords) Swap(i, j int) {
 	rs[i], rs[j] = rs[j], rs[i]
 }
 
+func validateIncrementalTxid(rs historyRecords, op OperationType) error {
+	if len(rs) == 0 {
+		return nil
+	}
+
+	lastTxid := rs[0].Txid
```

Comment:
```suggestion
	lastTxid := 0
```

Reply:
If we follow this way, then we also need to add a check whether `lastTxid == 0` or `i == 0`  inside the loop; we need a if check either way.  

Note the txid doesn't start from 0, and it can be any value in this case.

Reply:
Simplified the implementation to ensure the `len(rs) > 0`.